ote_cli/ote_cli/utils/tests.py

```diff
@@ -566,3 +566,19 @@ def xfail_templates(templates, xfail_template_ids_reasons):
                 "More than one reason for template. If you have more than one Jira tickets, list them in one reason."
             )
     return xfailed_templates
+
+
+def ote_explain_testing(template, root, ote_dir, args):
```

Comment:
In addition to it this sanity check. Can we also have a test which will check the correctness of the output saliency map (for some data_point-model use case) created by each explain method?

Reply:
Any update on this?

Reply:
updated to check output saliency map correctness.